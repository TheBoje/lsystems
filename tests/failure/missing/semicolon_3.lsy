DERIVATION: 3;
ANGLE_FACTOR: 60.0;
SCALE_FACTOR: 1.0;
AXIOM:F
IGNORE:A;

PRODUCTIONS:
    * < F > * --> FF;
END_PRODUCTIONS;
