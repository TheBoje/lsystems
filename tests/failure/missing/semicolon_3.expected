syntax error: expected ";" before "IGNORE:": line 5, column 1 to line 5, column 7:
IGNORE:A;
^^^^^^^

